src/fontra/views/editor/panel-selection-info.js

```diff
@@ -138,6 +138,11 @@ export default class SelectionInfoPanel extends Panel {
     this.haveInstance = !!instance;
 
     const selectedGlyphInfo = this.sceneController.sceneModel.getSelectedGlyphInfo();
+    const varGlyphController =
+      await this.sceneController.sceneModel.getSelectedVariableGlyphController();
+    const glyphLocked = varGlyphController
```

Comment:
I _think_ you could simplify to:
```js
    const glyphLocked = !!varGlyphController?.glyph.customData.locked;
```
ie. make sure you account for `varGlyphController` possibly being undefined, but once it isn't, customData is guaranteed to exists.

Btw. you should use a "fontra." prefix for the customData key. I suggest `fontra.glyph.locked`. We don't have a good place for constants yet, so for now just use a string.